packages/netlify-cms-core/src/actions/editorialWorkflow.ts

```diff
@@ -123,37 +121,30 @@ function unpublishedEntriesFailed(error: Error) {
   };
 }
 
-function unpublishedEntryPersisting(
-  collection: Collection,
-  entry: EntryMap,
-  transactionID: string,
-) {
+function unpublishedEntryPersisting(collection: Collection, entry: EntryMap) {
   return {
     type: UNPUBLISHED_ENTRY_PERSIST_REQUEST,
     payload: {
       collection: collection.get('name'),
       entry,
```

Comment:
I think we can pass only the `slug` here.
See https://github.com/netlify/netlify-cms/pull/4289/files#diff-c04b7dd92be649549b8b8806542af69cR76